tasking/pulp/tasking/celery_app.py

```diff
@@ -69,6 +69,8 @@ def initialize_worker(sender, instance, **kwargs):
     :param kwargs:   Other params (unused)
     :type  kwargs:   dict
     """
+    from pulp.tasking.services.worker_watcher import delete_worker
```

Comment:
I suggest we go with this for now, and can try to improve it later if we find a better approach.